meetings/http/2017-03-14/minutes.md

```diff
@@ -1,24 +1,65 @@
 # HTTP stream meeting 2
 
-### Time: Wednesday March 15th 
-* 6pm CET 
-* 5pm UK 
-* 1pm EST 
-* 10am PST
-            
-            
-### WebEx and Agenda/Minutes doc:
-            
-WebEx:      URL:        https://ibm2.webex.com/ibm2-en/j.php?MTID=m59d2772a43f192bf64ae3bdbd5c77faf
-            password:   swift-http
-        
-Agenda:     https://docs.google.com/document/d/1GTm7mJH2vCAK-Ts4M3t8C3boJ18XqiFnCRuyp55eKJE/edit?usp=sharing
-
-If you'd like to add anything to the agenda ahead of time, please update the agenda document in Google Docs.
 
 ### Initial Agenda:
-* Review of draft HTTP API Proposal 
+* Review of draft HTTP API Pitch 
+  * https://docs.google.com/document/d/1ErHA8RXLiTGvOPHNY0KtbWCvwR1IUz_2ffi7CxCUo_4/edit
 * Discussion of key points:
   * Use of URLRequest/Response
   * Value Types vs. Reference Types
 
+### Attendees:
+* Chris Bailey
+* Ben Cohen
+* Carl Brown
+* Stevey Brown
+* Tony Parker
+* Gregor Milos
+* Shmuel Kallner
+* Alex Blewitt
+* Allen
+* Chuck Goddard
+* George
+* Johannes Weiss
+* Ludovic Dewailly
+* Marcin Kliks
+* Nasser Ebrahim
+* Steve Algernon
+* Michael Papp
+* Thomas Catterall
+
+### Minutes:
+
+_Use of value types vs. reference types for request and response_ 
+* General consensus is in favour of using a value types for request and response.
+  * The argument that the should be references as they are passed around and mutated really don’t hold
+  * It should be noted that the reason why there is still NSURLRequest/Response (reference types) was because a subclass exists, rather than the intend that they should be reference types.
+  * The use of value types will help with reference cycles in closures (like middlewares) which you otherwise need to use weak self for.
+  * Should also help with reducing some of the ARC overhead
+* ARC overhead won’t necessarily be removed if the struct/value types has references inside it - and the body data is really a stream which has to be a reference type
```

Comment:
The approach would be to make use of `inout`, because ownership stays with the original request handler but middlewares need access to it.

I agree - post-prototyping we'll be able to validate the approach through benchmarking against a number of existing parses (that use reference types today).